alacritty_terminal/src/config/mod.rs

```diff
@@ -121,9 +121,11 @@ impl Default for EscapeChars {
 #[derive(Deserialize, Copy, Clone, Debug, PartialEq)]
 pub struct Cursor {
     #[serde(deserialize_with = "failure_default")]
-    pub style: CursorStyle,
+    pub style: ConfigCursorStyle,
     #[serde(deserialize_with = "option_explicit_none")]
-    pub vi_mode_style: Option<CursorStyle>,
+    pub vi_mode_style: Option<ConfigCursorStyle>,
+    #[serde(deserialize_with = "failure_default")]
+    pub blink_interval: u64,
```

Comment:
Do we really want to keep `u64` here? I don't see any reason to use it over something like `u32` or `u16` for blinking.

Reply:
There is a reason: The duration conversion takes a u64. So one way or another we need to convert it to u64 anyways at the only place we use it. So the best option is to just define it as u64 to begin with.

I see no benefit to define it as u16 when we then convert it later anyways. It's not like it's saving us a measurable amount of memory.